main/com.spotify/index.js

```diff
@@ -1318,7 +1322,8 @@ module.exports = class SpotifyDevice extends Tp.BaseDevice {
         try {
             await this.http_put_default_options(pauseURL, '');
         } catch (error) {
-            throwError('disallowed_action');
+            // throwError('disallowed_action');
+            console.error(error, error.stack);
```

Comment:
No, this one should remain as is, because we can show the error to the user if necessary. Only the release callback in the audio controller should swallow the error.